app/Views/errors/html/error_exception.php

```diff
@@ -90,7 +90,7 @@
                                         <?php
                                         $params = null;
                                         // Reflection by name is not available for closure function
-                                        if (substr($row['function'], -1) !== '}') {
+                                        if (!str_ends_with($row['function'], '}')) {
```

Comment:
Why is there no space after `!`?

Reply:
that should be a coding standard tool for that :)

Reply:
Yes, but I ran php-cs-fixer after running rector.

Reply:
I am not sure about that

Reply:
Ah, it is a HTML file, so php-cs-fixer does not fix?

Reply:
I believe we have Views in the ignore path, right?